llvm/lib/Target/SPIRV/SPIRVUtils.cpp

```diff
@@ -298,7 +287,7 @@ static bool isNonMangledOCLBuiltin(StringRef Name) {
          Name == "__translate_sampler_initializer";
 }
 
-std::string isOclOrSpirvBuiltin(StringRef Name) {
+std::string mayBeOclOrSpirvBuiltin(StringRef Name) {
```

Comment:
The name is misleading, I'd expect this function to return `bool` based on its name. And I don't like 'mayBe' as well, I suppose the result should be definite, or pls add a comment clarifying what's this about

Reply:
Ok, probably a more appropriate name is `getOclOrSpirvBuiltinDemangledName()`. The comment in SPIRVUtils.h should be also corrected.